archetype/src/main/resources/archetype-resources/README.md

```diff
@@ -7,7 +7,17 @@ With Storm installed, you must first generate an uberjar:
 mvn clean package
 ```
 
-before submitting the topology using the storm command:
+You need to have an instance of URLFrontier running. See https://github.com/crawler-commons/url-frontier/tree/master/service for instructions; the easiest way is to use Docker.
+You will also need to inject URLs into URLFrontier, using the [client](https://github.com/crawler-commons/url-frontier/tree/master/client). Fortunately, it is added as a dependency to this project so all
+you need to do is
+
+``` sh
+java -cp  target/${artifactId}-${version}.jar crawlercommons.urlfrontier.client.Client PutURLs -f seeds.txt
+```
+
+where _seeds.txt_ is a file containing URLs to inject, with one URL per line.
+
+You can now submit the topology using the storm command:
```

Comment:
I am wondering, if we should mention, that the user needs a local storm installation for this to work or do we just assume, that a person generating a stormcrawler project via the archetype **knows** that he/she needs a local storm environment setup and link to some reference?

Reply:
It says further up in the README 

> With Storm installed, you must first generate an uberjar:

I'll add a  link to the [Setting-up-a-Storm-cluster](https://storm.apache.org/releases/2.4.0/Setting-up-a-Storm-cluster.html)